Contracts/Grains/Portfolio/IPortfolio.cs

```diff
@@ -8,9 +8,11 @@ namespace CommunAxiom.Commons.Client.Contracts.Portfolio
 {
     public interface IPortfolio: IGrainWithStringKey
     {
-        Task<string> TestGrain(string Grain);
-        Task<string> CreatePortfolio(PortfolioDetails portfolio);
-        Task<PortfolioDetails> GetDetails();
-        Task<bool> IsSet();
+        Task<string> CreatePortfoliosList();
```

Comment:
Since grains will contain validation logic, return values for write operations should be encapsulated in CommunAxiom.Commons.Client.Contracts.OperationResult<T> objects